autogen/docs/behaviorspace.md.mustache

```diff
@@ -9,6 +9,23 @@ This guide has three parts:
 - [**Advanced Usage**](#advanced-usage): How to use BehaviorSpace from the command
   line, or from your own Java code.
 
+A number of new features were introduce in NetLogo 6.4.0:
+
+- [**Subexperiment syntax**](#subexperiment-syntax): A syntax for allowing parameter
+ combinations to be run separately, rather than being expanded combinatorically.
+- [**Run metrics when**](#run-metrics-when): The recording of measurements can now 
```

Comment:
I would prefer something like "Measurements can now be conditionally recorded, controlled by a reporter." It's not entirely clear with the current wording how exactly the reporter would control the recording of measurements. It could be interpreted as "reporters can alter the recorded measurements" or something like that.